backend/models/office_hours/oh_event_status.py

```diff
@@ -0,0 +1,26 @@
+"""Represent the status of an office hours event."""
+
+from pydantic import BaseModel
+from typing import Sequence
+
+from oh_ticket import OfficeHoursTicket
+
+__authors__ = ["Sadie Amato, Bailey DeSouza, Meghan Sun, Maddy Andrews"]
+__copyright__ = "Copyright 2024"
+__license__ = "MIT"
+
+
+class TAOfficeHoursEventStatus(BaseModel):
+    """Pydantic model to represent the status of a given event, including tickets in the queue, for a given TA."""
+
+    active_tickets: Sequence[OfficeHoursTicket]
+    num_open: int
+    num_pending: int
+
+
+class StudentOfficeHoursEventStatus(BaseModel):
+    """Pydantic model to represent the status of a given event, including their position in the queues, for a given student."""
+
+    num_open: int
+    num_pending: int
+    position: int
```

Comment:
Would this ultimately be a field in the ticket model, or would this be queried separately? If queried separately, it might be good to add a reference to a ticket here!

Reply:
The ticket would be queried separately! This is a model for fetching information about the current status of all tickets that are active/ in the queue for an office hours event. 

<img height=300 src="https://github.com/unc-csxl/csxl.unc.edu/assets/69722735/d9fd1930-e8b7-45c6-abb7-a44e967ff3c7" />

Reply:
The `StudentOfficeHoursEventStatus` class represents a model related to an API response we want to make, so it is separate and not related to our ticket model. It's meant to represent a summary of all ticket's status in a queue!

We also didn't want to have a ticket reference here since the information in this model would likely update more often as tickets are being made, called, and closely more frequently when a user-created ticket would be updated. 

This image is what we intended `StudentOfficeHoursEventStatus` class API response model to represent: 

<img src="https://github.com/unc-csxl/csxl.unc.edu/assets/69722735/698fda9f-0b57-4989-a981-244220eea8c1" width=300 >